tests/validation/p1_run_test.go

```diff
@@ -0,0 +1,94 @@
+package validation
+
+import (
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/rancher/rio/tests/testutil"
+)
+
+func p1RunTests(t *testing.T, when spec.G, it spec.S) {
```

Comment:
I'm confused by this, I thought are integration tests were making up the p1 test cases ? This is mostly duplicated. Duplication is bad because it will increase our drone and general testing run time.